bitnami/grafana-operator/values.yaml

```diff
@@ -102,6 +102,9 @@ operator:
     enabled: true
     runAsUser: 1001
     runAsGroup: 1001
+    capabilities:
+      drop:
+      - ALL
```

Comment:
Why are we dropping all capabilities when the containers are running by default as non-root and also setting the `allowPrivilegeEscalation` to false?

Reply:
I just added it to comply with our policies which ask for that. It is not needed and if I should not add it we can drop it from the `securityContext`

Reply:
Then in this case I would remove it just to keep the PR only related to the configurability improvements. Thank you very much for your work!

Reply:
Should i bump then only the patch release?
This is not changing the behaviour of the `values.yaml` interface

Reply:
Yes, just bump the patch version. Thank you!